websites/V/Vidstreaming/presence.ts

```diff
@@ -23,32 +27,35 @@ presence.on("iFrameData", (data) => {
   }
 });
 
-function getTimestamps(videoTime, videoDuration): any {
-  var startTime = Date.now();
-  var endTime = Math.floor(startTime / 1000) - videoTime + videoDuration;
-  return [Math.floor(startTime / 1000), endTime];
-}
-
 presence.on("UpdateData", async () => {
-  const info = await presence.getSetting("sSI");
-  const elapsed = await presence.getSetting("sTE");
-  const videoTime = await presence.getSetting("sVT");
+  const info = await presence.getSetting("sSI"), 
+  elapsed = await presence.getSetting("sTE"), 
+  videoTime = await presence.getSetting("sVT");
   if (videoTime) {
-    if (lastPlaybackState != playback) {
+    if (lastPlaybackState !== playback) {
       lastPlaybackState = playback;
-      browsingStamp = Math.floor(Date.now() / 1000);
+      presence.info("Video Time is On");
     }
   }
+  else{
+    lastPlaybackState = null;
+    presence.info("Video Time is Off");
+  }
   if (elapsed) {
     browsingStamp = Math.floor(Date.now() / 1000);
-    console.log("Elapsed is on");
+    presence.info("Elapsed is On");
+  }
+  else{
+    browsingStamp = null;
+    presence.info("Elapsed is Off");
   }
```

Comment:
if you have this, don't set browsingStamp before then? and just don't set it to null